src/main/java/uk/gov/hmcts/divorce/document/content/templatecontent/FinalOrderCanApplyTemplateContent.java

```diff
@@ -54,9 +55,15 @@ public Map<String, Object> getTemplateContent(CaseData caseData, Long caseId, Ap
         templateContent.put(IS_JOINT, !caseData.getApplicationType().isSole());
         templateContent.put(FINAL_ORDER_OVERDUE_DATE, caseData.getFinalOrder().getDateFinalOrderEligibleFrom().plusMonths(12)
                 .format(getDateTimeFormatterForPreferredLanguage(applicant.getLanguagePreference())));
-        templateContent.put(PARTNER, commonContent.getPartner(caseData, caseData.getApplicant2(),
-                caseData.getApplicant1().getLanguagePreference()));
+        templateContent.put(PARTNER, getPartnerInfo(caseData,applicant,applicant.getLanguagePreference()));
 
         return templateContent;
     }
+
+    private String getPartnerInfo(CaseData caseData, Applicant applicant, LanguagePreference languagePreference) {
+        if (applicant.equals(caseData.getApplicant1())) {
+            return commonContent.getPartner(caseData, caseData.getApplicant2(), languagePreference);
+        }
+        return commonContent.getPartner(caseData, caseData.getApplicant1(), languagePreference);
+    }
```

Comment:
There is a method in CommonContent class with LanguagePreference as parameter in case you want to reuse that?